x/ecocredit/client/basket/tx.go

```diff
@@ -0,0 +1,78 @@
+package basket
+
+import (
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+
+	keeper "github.com/regen-network/regen-ledger/x/ecocredit/basket"
+)
+
+const (
+	FlagRetirementLocation = "retirement-location"
+	FlagRetireOnTake       = "retire-on-take"
+)
+
+func TxTakeFromBasket() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "take-from-basket [owner] [basket_denom] [amount]",
+		Short: "takes credits from a basket",
+		Long: strings.TrimSpace(`takes credits from a basket starting from the oldest credits first.
+
+Parameters:
+		owner: account address of the owner of the basket.
+		Note, the'--from' flag is ignored as it is implied from [owner]
```

Comment:
Why not removing `owner` param from this command, and use `--from` instead? Sounds to be less hacky.